8.7-alpha5 release notes (#5136)
camunda/camunda-docs: docs/reference/release-notes/870.md

- Reviews cannot be performed by the user who created the process application version, and the reviewer is logged in the version history of a process application
- Admins can enable production deployments for reviewed process applications as an alternative to using their own deployment pipeline

For more information, see the [process governance](/docs/components/modeler/web-modeler/process-application-pipeline.md) documentation.

<!-- https://github.com/camunda/product-hub/issues/2583 -->

### BPMN-to-text in BPMN Copilot

Besides [generating BPMN](/docs/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md), Camunda's BPMN Copilot now supports generating text from the BPMN diagram. This can be used to:

- Rapidly draft process documentation
- Learn how the process works
- Explain the process's behavior to stakeholders

For more information, see the [BPMN Copilot documentation](/components/early-access/alpha/bpmn-copilot/bpmn-copilot.md).

<!-- https://github.com/camunda/product-hub/issues/2576 -->

### Advanced configuration options for Camunda 8 Run

> The Camunda 8 Run startup script now supports additional configuration parameters: a web application port, the location of keystore TLS certificates, and log level. This release also introduces a new `--docker` option, which starts C8Run with the `docker-compose up` command, and deploys Camunda 8 in containers instead of starting with a Java engine. For more information, see the [Camunda 8 Run documentation](/docs/self-managed/setup/deploy/local/c8run.md).

Reviewer: ```suggestion
The Camunda 8 Run startup script now supports additional configuration parameters: a web application port, the location of keystore TLS certificates, and log level. A new `--docker` option also allows you to start C8Run with the `docker-compose up` command, and deploy Camunda 8 in containers instead of starting with a Java engine. For more information, see the [Camunda 8 Run documentation](/docs/self-managed/setup/deploy/local/c8run.md).
```
Rewording suggestion

<!-- https://github.com/camunda/product-hub/issues/2459 -->

### Health and capacity monitoring for clusters in Console

> This release introduces a real-time cluster capacity metric for monitoring and scaling your Camunda Platform 8 clusters.

Reviewer: ```suggestion
This release introduces a real-time cluster capacity metric for monitoring and scaling your Camunda 8 clusters.
```

<!-- https://github.com/camunda/product-hub/issues/2346 -->

> ### Document extraction for unstructured data

Reviewer: I think this is IDP, so probably could do with some more content - I can take a look at this entry on Monday

Author: Thanks! The only epic related to this is https://github.com/camunda/product-hub/issues/2521, and I haven't heard anything in the request for reviews in product. Would love to expand on this/link to docs as they're available.

Reviewer: I am finalising this and pushing the docs out tomorrow hopefully, so can add the link etc then

The new document extraction UI allows users to configure projects for unstructured (LLM extraction) data. Define the taxonomy, test cases, and choose models for LLM-based extraction, then publish and share projects across the organization.

<!-- https://github.com/camunda/product-hub/issues/2521 -->

### Amazon Web Services (AWS) region support <span class="badge badge--long" title="This feature affects SaaS">SaaS</span>

> Camunda 8 SaaS now supports Amazon Web Services (AWS) deployments. When [creating a new cluster](/components/console/manage-clusters/create-cluster.md), you can select from the following new AWS regions:

Reviewer: I would probably add the regions page lonk rather than the creating a new cluster one as it is more relevant.
